### DISCUSSION TITLE
Implementing the AmIPwned funcitonality

### DISCUSSION THREAD

**Discussion by p3rcy** [zxcvbn-ts/zxcvbn#110] [Q&A]
I'd like to debounce as i'm planning on showing a password strength meter on password input - I see that the documentation recommends that I use the included debounce helper, but I can't find an example of usage.

Could I get some help here please? 

Thanks in Advance (Great repo by the way! 😄 )

**Comment by MrWook** [comment]
Hey sorry for the late reply.
You are correct the debounce example is missing in the documentation. I will add it if i have a correct internet connection again :D 
But the demo page is using it. Maybe it will help you. Here is a vue component of it https://github.com/zxcvbn-ts/zxcvbn/blob/master/docs/.vuepress/components/ZxcvbnInteractive.vue

It's basically 
```
import {
  zxcvbnAsync,
  debounce,
} from '@zxcvbn-ts/core'

let result
function someCallableFunction (){
  ...do your magic for example get the value from an input field or somewhere else
  const value = getInputValue()
  result = this.zxcvbnAsync(value)
}

const debouncedZxcvbn = debounce(someCallableFunction, 200)
```

Kinda like this

**Comment by MrWook** [comment]
I added some examples: 
https://zxcvbn-ts.github.io/zxcvbn/guide/examples/#debounce
